etc/DependencyInstaller.sh

```diff
@@ -126,8 +126,8 @@ _installOrTools() {
     os=$1
     version=$2
     arch=$3
-    orToolsVersionBig=9.4
-    orToolsVersionSmall=${orToolsVersionBig}.1874
+    orToolsVersionBig=9.5
+    orToolsVersionSmall=${orToolsVersionBig}.2237
```

Comment:
Is this change required to add support for ubuntu 22.10? If it is not required I would like to have a different PR for this, since it will require me to update the version of or-tools on our CI to keep up with the installer.

Reply:
@vvbandeira it is required. Ubuntu 22.10 support was added in the [or-tools v9.5 release](https://github.com/google/or-tools/releases/tag/v9.5)